### PR TITLE
feat: Prettier on CI

### LINKED ISSUE #409: Integrate Prettier on CI
**Problem**
GraphQL, YAML and JSON files remain unformatted formatted inconsistently

**Solution**

Integrate [prettier](https://prettier.io/) on the CI as a lint check to make sure all the .graphql files and yaml and json files are consistently formatted.

### PR DESCRIPTION
**Pull request by kremedev**
**Summary:**  
/claim #409

**Issue Reference(s):**  
Fixes #409

**Build & Testing:**

- [x] I ran `cargo test` successfully.
- [x] I have run `./lint.sh` to address and fix linting issues.

### CODE REVIEW

**Comment by tusharmath** [comment]
@kremedev The build is failing. I think the integration tests are whitespace sensitive.

File: .prettierignore [hunk 1]
```diff
@@ -0,0 +1,5 @@
+tests/graphql/inline-many.graphql
+tests/graphql/inline-many-list.graphql
+tests/graphql/test-inline.graphql
+tests/graphql/test-inline-list.graphql
+tests/graphql/test-server-vars.graphql
```
Comment (tusharmath) [inline .prettierignore:5]: Why are we ignoring these files?

File: .prettierrc [hunk 2]
```diff
@@ -0,0 +1,14 @@
+{
+  "semi": false,
+  "singleQuote": false,
+  "printWidth": 80,
```
Comment (tusharmath) [inline .prettierrc:4]:
```suggestion
  "printWidth": 120,
```

File: .prettierignore (hunk 1 above)
Comment (kremedev) [reply .prettierignore:5]: Prettier puts space between array elements in these files and I couldn't find any way to change this action other than ignoring it. I tried changing the actual and expected logic for the test but I couldn't figure it out either :(

**Comment by amitksingh1490** [comment]
@kremedev this PR solves your spacing issue feel free to take this code and update your branch to pass the build
https://github.com/tailcallhq/tailcall/pull/433

**Comment by kremedev** [comment]
I knew it would be solved in this file, but I couldn't succeed like you. Thank you @amitksingh1490

**Review by amitksingh1490 — APPROVED** [review]

**Pull request merged**